msg/sensor_correction.msg

```diff
@@ -6,52 +6,21 @@ uint64 timestamp		# time since system start (microseconds)
 
 # Corrections for gyro angular rate outputs where corrected_rate = raw_rate * gyro_scale + gyro_offset
 # Note the corrections are in the sensor frame and must be applied before the sensor data is rotated into body frame
-# corrections for uORB instance 0
-float32[3] gyro_offset_0	# gyro XYZ offsets in the sensor frame in rad/s
-float32[3] gyro_scale_0		# gyro XYZ scale factors in the sensor frame
-
-# corrections for uORB instance 1
-float32[3] gyro_offset_1	# gyro XYZ offsets in the sensor frame in rad/s
-float32[3] gyro_scale_1		# gyro XYZ scale factors in the sensor frame
-
-# corrections for uORB instance 2
-float32[3] gyro_offset_2	# gyro XYZ offsets in the sensor frame in rad/s
-float32[3] gyro_scale_2		# gyro XYZ scale factors in the sensor frame
+uint32[3] gyro_device_ids
+float32[3] gyro_offset_0	# gyro 0 XYZ offsets in the sensor frame in rad/s
+float32[3] gyro_offset_1	# gyro 1 XYZ offsets in the sensor frame in rad/s
+float32[3] gyro_offset_2	# gyro 2 XYZ offsets in the sensor frame in rad/s
```

Comment:
So no more gyro scale calibration?

Reply:
There is no gyro scale calibration (TC or regular calibration), but see the accel comment below for more detail.